Source/VSProps/QtCompile.props

```diff
@@ -88,6 +90,10 @@
   <!--Copy the needed dlls-->
   <ItemGroup>
     <QtLibNames Include="Qt5Core$(QtLibSuffix);Qt5Gui$(QtLibSuffix);Qt5Widgets$(QtLibSuffix)" />
+
+    <!-- Include ANGLE for ARM64 since there is no native OpenGL -->
+    <QtLibNames Condition="'$(Platform)'=='ARM64'" Include="$(QtLibNames);libEGL$(QtLibSuffix);libGLESv2$(QtLibSuffix)" />
```

Comment:
I only had to place libGLESv2, will libEGL be used in some case?

Reply:
If we want to use it for the software rasterizer or GL backend, potentially